oca_dependencies.txt

```diff
@@ -1,3 +1,5 @@
 server-tools
 product-attribute
 server-ux
+hr
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 11.0-mig-purchase_request
```

Comment:
dependency merged, can you remove this and rebase?

Reply:
done!